backend/de.metas.material/dispo-commons/src/main/java/de/metas/material/dispo/commons/repository/atp/AvailableToPromiseResultBuilder.java

```diff
@@ -105,24 +103,7 @@ public AvailableToPromiseResult build()
 
 	public void addQtyToAllMatchingGroups(@NonNull final AddToResultGroupRequest request)
 	{
-		boolean addedToAtLeastOneGroup = false;
-		for (final AvailableToPromiseResultBucket bucket : buckets)
-		{
-			if (bucket.addQtyToAllMatchingGroups(request))
-			{
-				addedToAtLeastOneGroup = true;
-			}
-		}
-
-		if (!addedToAtLeastOneGroup)
-		{
-			throw new AdempiereException("No matching group found for AddToResultGroupRequest")
-					.appendParametersToMessage()
-					.setParameter("request", request)
-					.setParameter("buckets", buckets)
-					.setParameter("this", this);
-		}
-
+		buckets.forEach(bucket -> bucket.addQtyToAllMatchingGroups(request));
```

Comment:
```suggestion
		// note that we might select more quantities than we actually wanted (bc of the way we match attributes in the query using LIKE)
		// for that reason, we need to be lenient in case not all quantities can be added to a bucked
		buckets.forEach(bucket -> bucket.addQtyToAllMatchingGroups(request));
```